### PR TITLE
try fix travis

### PR DESCRIPTION
**Pull request by KostyaSha**
This change is [Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/148)

### CODE REVIEW

**Comment by KostyaSha** [comment]
@lanwen

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
+sudo: required
+dist: trusty
 language: java
 jdk: oraclejdk8
 before_install:
  - pip install --user codecov
-install: travis_wait mvn install
+script: mvn package | grep -v Downloading
```
Comment (lanwen) [inline .travis.yml]: should be 'Download'

**Comment by lanwen** [comment]
migrated to travis-test branch

**Comment by KostyaSha** [comment]
Can't test via PR because you disabled PR builds.

**Comment by lanwen** [comment]
Yep, branch test works fine.

**Pull request closed without merging**